deployments/helm-charts/wso2-ai-agent-management-platform/values.yaml

```diff
@@ -88,7 +88,9 @@ agentManagerService:
     authHeader: "Authorization"
     dbOperationTimeout: 30
     healthCheckTimeout: 5
-    corsAllowedOrigin: "*"
+    corsAllowedOrigin: "http://localhost:3000"
```

Comment:
Adding to this config is not quite right I believe. Because, these are not for the amp-service, but for the agent service that is being deployed. 

Shall we add these to a separate config section? 

agent-workload or similar? 

No back-end code needs to be changed.